packages/mobile/src/account/Settings.tsx

```diff
@@ -329,6 +329,24 @@ export class Account extends React.Component<Props, State> {
     this.setState({ showRevokeModal: false })
   }
 
+  goToChangePin = () => {
+    ValoraAnalytics.track(SettingsEvents.change_pin_start)
+    ensurePincode()
+      .then((pinIsCorrect) => {
+        if (pinIsCorrect) {
+          ValoraAnalytics.track(SettingsEvents.change_pin_current_pin_entered)
+          this.props.navigation.navigate(Screens.PincodeSet, {
+            isVerifying: false,
+            changePin: true,
+          })
+        }
+      })
+      .catch((error) => {
+        ValoraAnalytics.track(SettingsEvents.change_pin_current_pin_error)
+        Logger.error('NavigationService@onPress', 'PIN ensure error', error)
+      })
+  }
```

Comment:
Nit: Can you please use the async/await with try/catch block pattern? I'm not sure if that's our official team preference but that's what we use most throughout the codebase